src/app/addcustomer/addcustomer.component.html

```diff
@@ -76,7 +76,25 @@ <h5 class="my-3">Add Customer</h5>
             placeholder="Enter Route Number"
           />
         </div>
-        <div class="col-sm-6 col-xs-12 mb-3">
+        <div class="col-sm-3 col-xs-12 mb-3">
+          <label for="customertype" class="label m-0"
+            >Select Customer Type</label
+          >
+          <select
+            name="customertype"
+            id="customertype"
+            class="form-control"
+            [(ngModel)]="customertype"
+            (change)="getCustomerTypeId()"
+          >
+            <option
+              value="{{ cust.customertypename }}"
```

Comment:
You could have used something like

name="customertype"
            id="customertype"
            class="form-control"
            [(ngModel)]="customertype"
          >
            <option
              value="{{ cust.customertypeid}}">